install-spack.sh

```diff
@@ -35,15 +28,10 @@ pocl_dir=$(spack find -p pocl| tail -1 | awk '{print $2}')
 export OCL_ICD_VENDORS=$pocl_dir/etc/OpenCL/vendors/
 echo 'export OCL_ICD_VENDORS=$pocl_dir/etc/OpenCL/vendors/' >> $HOME/.bashrc
 
-echo "==== Installing pip packages"
-pip install wheel pyvisfile numpy
-
-for module in dagrt leap loopy meshmode grudge mirgecom; do (cd $module && pip install -e .) ; done
-EOF
-
+./install-pip.sh
```

Comment:
`install-pip.sh` no longer exists. :) Though maybe it should? And be called by both `install-spack.sh` and `install-dependencies.sh`? (More scripts, yay!)

Reply:
Good catch. I forgot to rename install-pip.sh to install-dependencies.sh here.